helix-term/src/commands/typed.rs

```diff
@@ -2291,6 +2321,20 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: force_write,
             signature: CommandSignature::positional(&[completers::filename]),
         },
+        TypableCommand {
+            name: "write-buffer-close",
+            aliases: &["wbc"],
+            doc: "Write changes to disk and closes the buffer. Accepts an optional path (:write some/path.txt)",
+            fun: write_buffer_close,
+            signature: CommandSignature::positional(&[completers::filename]),
+        },
+        TypableCommand {
+            name: "write-buffer-close!",
+            aliases: &["wbc!"],
+            doc: "Force write changes to disk creating necessary subdirectories and closes the buffer. Accepts an optional path (:write some/path.txt)",
```

Comment:
The docs here should mention ":write-close-buffer(!)" instead. It looks like force_write could also be updated to say ":write!" instead of ":write" but that isn't totally related to this PR

Reply:
Right, thanks for noticing. Fixed the `write!` as well